src/I18n/I18n.php

```diff
@@ -246,6 +238,20 @@ public static function defaultLocale()
     }
 
     /**
+     * Set default fallback domain. If a domain can't provide a translation
+     * messages from the fallback domain are used.
+     *
+     * Get the currently set domain if argument is null.
+     *
+     * @param string|null $name name of the fallback-translator
+     * @return string
+     */
+    public static function defaultFallbackDomain($name = null)
```

Comment:
> Ok, I don't think we need to customize the fallback domain though.

Right we don't. I was responding to @Schlaefer's comment of dropping fallback domain altogether.